tests/test_views.py

```diff
@@ -0,0 +1,59 @@
+import json
+
+from .db_test import db
+from .factories import AnimeFactory
+
+
+def test_animes_route(client, create_default_anime):
+    AnimeFactory(
+        anime_id=111222,
+        name='anime_1',
+        genre='Horror',
+        type='TV',
```

Comment:
Parameters similar you the default ones doesn't need to be specified => type, episodes

Reply:
If you added those to make it more comprehensive with the expected result, i am ok to keep those type, episodes here